### PR TITLE
Fix mismatches in INSTALL.md

### PR DESCRIPTION
**Pull request by dr-pain**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -75,18 +79,24 @@ Required Software:
```
Comment (napetrov) [inline INSTALL.md]: Could you please also update one line above
from:
scripts/mklfpk.sh [32|32e]

to 
scripts/mklfpk.sh

File: INSTALL.md [hunk 2]
```diff
@@ -71,22 +75,28 @@ Required Software:
...
 6. Download and install Intel(R) Threading Building Blocks (Intel(R) TBB):
 
-    - **Windows\***:
-        - Download and install free Community License Intel TBB (see [Get Intel(R) Performance Libraries for Free](https://registrationcenter.intel.com/en/forms/?productid=2558&licensetype=2)) or build your own Intel TBB from [Intel(R) TBB GitHub repository](https://github.com/intel/tbb).
-        - Set the environment variables for Intel TBB. For example:
+    Download and install free Community License Intel(R) TBB (see [Get Intel(R) Performance Libraries for Free](https://registrationcenter.intel.com/en/forms/?productid=2558&licensetype=2)).
+    Set the environment variables for for Intel(R) TBB. For example:
 
-            call "C:\Program Files (x86)\IntelSWTools\compilers_and_libraries\windows\tbb\bin\tbbvars.bat" intel64 all
+    - oneTBB (Windows\*):
```
Comment (outoftardis) [inline INSTALL.md]: Why it's oneTBB here, but Intel(R) TBB 2020 on the line 89? Is it supposed to be like this?

Comment (dr-pain) [reply INSTALL.md]: It is "usage examples". We can`t list all possible variants.

File: INSTALL.md [hunk 3]
```diff
@@ -71,22 +75,28 @@ Required Software:
 
     - **Linux\***:
 
-            scripts/mklfpk.sh [32|32e]
+            scripts/mklfpk.sh
```
Comment (akhekhn) [inline INSTALL.md]: For Linux command should be : "./scripts/mklfpk.sh"

**Pull request merged**